<launch>
    <arg name="estimator_config" default="$(find ros_estimators)/config/pacejka_car_ekf.yaml" />
    <arg name="input_topic" default="control_input" />
    <arg name="vicon_topic" default="vicon" />
    <arg name="imu_topic" default="imu" />
    <arg name="state_est_topic" default="estimation_node/best_state" />

    <arg name="world_frame" default="world" />
    <arg name="track_frame" default="world" />
    <arg name="update_track_transform" default="false" />

    <node pkg="ros_estimators" name="estimation_node" type="ros_estimation_node" output="screen">
        <rosparam command="load" file="$(arg estimator_config)" />
        <remap from="imu" to="$(arg imu_topic)" />
        <remap from="vicon" to="$(arg vicon_topic)" />
        <remap from="control_input" to="$(arg input_topic)" />
        <remap from="state_est" to="$(arg state_est_topic)" />

        <param name="world_frame" type="str" value="$(arg world_frame)"></param>
        <param name="track_frame" type="str" value="$(arg track_frame)"></param>
        <param name="update_track_transform" type="bool" value="$(arg update_track_transform)"></param>
    </node>

</launch>
